### ISSUE TITLE
When load model, it seems doesn't work right.

### ISSUE THREAD

**Issue by LaiPiXiong** [pytorch/pytorch#5686]
I used to test model after training, it's test accuracy is about 0.68. Then i write a 'test.py' code, and i test the mode on the same test data, but the accuracy is about 0.12. 
I load the model as follows:
`featureModel = torch.load('model/ImageModel.pkl)
featureModel.eval()
captionmodel = torch.load('model/CaptionModel.pkl)
captionmodel.eval()`
What's wrong may it be?

**Comment by apaszke** [comment]
We're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).

**Comment by apaszke** [comment]
It sometimes happens with BatchNorm modules, if the statistics on the training set don't match those in test set (e.g. because they haven't been trained for long enough)

**Comment by LaiPiXiong** [comment]
@apaszke But I test on the same data( these two ways test on the same data with different results). I guess I do not load model in the right way. Any problem in my code?

**Comment by LaiPiXiong** [comment]
@apaszke I use same test data in two different test ways, which leads to different results. So i think the model is stable enough. I guess maybe i do not load model in right way.

**Comment by LaiPiXiong** [comment]
@apaszke The Link you give is "Not Found". Any other Links?

**Comment by apaszke** [comment]
Sorry, should be ok now
